### ISSUE TITLE
Better Alternate Attributes for Images

### ISSUE THREAD

**Issue by dhershman1** [dhershman1/dusty-codes-site#6]
I should go through and make sure each image that is used also has a alternative attribute, as well as making sure that this alt text is informative to what the image is suppose to be.

Making sure that even if the content isn't there we still know what it is, or what it should be.
